addons/mail/static/src/widgets/discuss_invite_partner_dialog/discuss_invite_partner_dialog.js

```diff
@@ -113,9 +113,11 @@ const PartnerInviteDialog = Dialog.extend({
             names
         );
         this.env.services['notification'].notify({
+            headerIconDisable: true,
             message: notification,
             title: this.env._t("New people"),
-            type: 'warning',
+            titleColor: '#212529',
```

Comment:
Isn't this color `$gray-900`? Would we best if we could use the SCSS variable here instead. Maybe giving full control to HTML color as param like we do currently is too flexible?

Reply:
Hello.. @seb-odoo 
As you suggested early to maintain "widget.styles" attribute in JS side and if we put "$gray-900" instead of "#212529" then at that time "$gray-900" will not be parsed 
![Screen_Shot_2020-11-03_at_11 41 29_AM](https://user-images.githubusercontent.com/54100921/97956475-bf024e80-1dce-11eb-978b-9e4f1f4cdf4f.png)
and will not give proper color.

Reply:
Indeed, it needs to be in SCSS to work, which makes it complicated to handle here (JS has no knowledge of SCSS variables unfortunately). I think that should be improved, but not in this PR.

I guess for now you can keep using `#212529` but at least create a single variable/field for it in JS so it is not copy/pasted multiple times.

Reply:
Hello @seb-odoo 
As for this suggestion i made variable "TITLE_COLOR" for "#212529" in FILE (/addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js) but in (/addons/mail/static/src/widgets/discuss_invite_partner_dialog/discuss_invite_partner_dialog.js) FILE "#212529" is used only once so i put it as it is.

Reply:
Move the variable somewhere accessible from everywhere to avoid having multiple occurrences.

What is common to all models/components is the `env`, so I guess it must be somewhere in there (exactly key name for this variable in `env` is to be confirmed later, but just try something that works for now)

If it's specific to mail I guess you'll have to add it in `main.js`